openslides/motions/views.py

```diff
@@ -134,14 +134,15 @@ def create(self, request, *args, **kwargs):
                 'title',
                 'text',
                 'reason',
+                'category_id',
```

Comment:
Are you that you want to allow users without managing perms to set the category of the new motion?

Reply:
Yes, I'm sure. It's a valid use case. We already discussed it on last code sprint meeting. - Maybe we can add a config later.